resources/fetching/fetchers/azure/assets_enricher_storage_account.go

```diff
@@ -0,0 +1,76 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I'm used as pattern to have the "type" prepended to the file name, not appeneded. So `storage_account_assets_enricher`. It reads a bit more natural to me.

I'll follow what you decide @moukoublen, but wanted to bring the point up just in case you agree with me :D

Reply:
Indeed. The "advantage" of using names with a hierarchy format is that you have a "grouped" sorting by default as a side effect. 

So having many `assets_enricher_***.go` will be sorted together makes it easier to navigate. 

I could change it, though, if that's not useful.